src/test/java/de/uhd/ifi/se/decision/management/jira/consistency/TestConsistencyCheckEventTrigger.java

```diff
@@ -0,0 +1,79 @@
+package de.uhd.ifi.se.decision.management.jira.consistency;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.event.issue.IssueEvent;
+import com.atlassian.jira.event.type.EventType;
+import com.atlassian.jira.issue.MutableIssue;
+import com.atlassian.jira.issue.comments.Comment;
+import com.atlassian.jira.issue.status.MockStatus;
+import com.atlassian.jira.mock.ofbiz.MockGenericValue;
+import com.atlassian.jira.user.ApplicationUser;
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.StatusClosedTrigger;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.WorkflowDoneTrigger;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraUsers;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestConsistencyCheckEventTrigger extends TestSetUp {
+	ConsistencyCheckEventTrigger trigger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)